CoreScriptsRoot/Modules/Settings/SettingsHub.lua

```diff
@@ -214,7 +223,7 @@ local function CreateSettingsHub()
 			Size = SETTINGS_SHIELD_SIZE,
 			Position = SETTINGS_SHIELD_ACTIVE_POSITION,
 			BorderSizePixel = 0,
-			ClipsDescendants = true,
```

Comment:
Do we need to be concerned about non-VR applications of this ClipsDescendants variable?